examples/protocol_specification_ex/fipa.yaml

```diff
@@ -0,0 +1,25 @@
+name: fipa
+author: fetchai
+version: 0.1.0
+license: Apache 2.0
+description: 'A protocol for FIPA ACL.'
+speech_acts:
+  cfp:
+    query: Query
+  propose:
+    proposal: list
+    price: float
+  accept_w_inform:
+    info: dict
```

Comment:
I am concerned that dict is not further specified. I think we should make it strongly typed.

Reply:
info is dict[str, Any]. The Any part is problematic. What should I change it to?